src/code/audio/audio_heap.c

```diff
@@ -846,23 +921,26 @@ s32 AudioHeap_ResetStep(void) {
 
 void AudioHeap_Init(void) {
     s32 pad1[4];
-    s16* mem;
-    s32 persistentMem;
-    s32 temporaryMem;
-    s32 totalMem;
-    s32 wantMisc;
+    s16* addr;
+    size_t persistentSize;
+    size_t temporarySize;
+    size_t cachePoolSize;
+    size_t miscPoolSize;
     u32 intMask;
     s32 i;
     s32 j;
     s32 pad2;
-    AudioSpec* spec;
+    AudioSpec* spec = &gAudioSpecs[gAudioContext.audioResetSpecIdToLoad]; // Audio Specifications
 
-    spec = &gAudioSpecs[gAudioContext.audioResetSpecIdToLoad];
     gAudioContext.sampleDmaCount = 0;
-    gAudioContext.audioBufferParameters.frequency = spec->frequency;
-    gAudioContext.audioBufferParameters.aiFrequency = osAiSetFrequency(gAudioContext.audioBufferParameters.frequency);
+
+    // audio buffer parameters
+    gAudioContext.audioBufferParameters.samplingFreq = spec->samplingFreq;
+    gAudioContext.audioBufferParameters.aiSamplingFreq =
+        osAiSetFrequency(gAudioContext.audioBufferParameters.samplingFreq);
+
     gAudioContext.audioBufferParameters.samplesPerFrameTarget =
-        ((gAudioContext.audioBufferParameters.frequency / gAudioContext.refreshRate) + 0xF) & 0xFFF0;
+        ((gAudioContext.audioBufferParameters.samplingFreq / gAudioContext.refreshRate) + 0xF) & 0xFFF0;
```

Comment:
It does more than remind me of `ALIGN16`, it's exactly the `ALIGN16` operation and the macro works just fine here. Nice catch.

Seems fine to me to use the macro, especially since that's what the code is doing. All it's saying is to make sure samples are processed in batches of 16 bytes at once, which is fairly standard